pkg/cluster/api/pdapi.go

```diff
@@ -614,23 +617,24 @@ func (pc *PDClient) DelStore(host string, retryOpt *utils.RetryOption) error {
 		}
 	}
 	if err := utils.Retry(func() error {
-		currStores, err := pc.GetStores()
+		currStore, err := pc.GetCurrentStore(host)
 		if err != nil {
 			return err
 		}
+		if currStore == nil {
```

Comment:
If we should check if currStore ==nil, I think it's better not return err in GetCurrentStore(host) while not found store